gitlab/assets/dashboards/overview.json

```diff
@@ -0,0 +1,419 @@
+                    "scale": "linear"
+                }
+            },
+            "time": {},
+            "title": true,
+            "title_align": "left",
+            "title_size": "16",
+            "title_text": "User Logins ",
+            "type": "timeseries",
+            "width": 31,
+            "x": 1,
+            "y": 49
+        },
+        {
+            "height": 13,
+            "id": 13,
+            "legend": false,
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:gitlab.unicorn.workers{*} by {host}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": "16",
+            "title_text": "Unicorn Workers",
+            "type": "timeseries",
+            "width": 43,
+            "x": 34,
+            "y": 45
+        },
+        {
+            "height": 13,
+            "id": 14,
+            "legend": false,
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "sum:gitlab.transaction.event_push_branch_total{*}.as_count()",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "bars"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": "16",
+            "title_text": "Branches Pushed",
+            "type": "timeseries",
+            "width": 43,
+            "x": 79,
+            "y": 13
+        },
+        {
+            "height": 13,
+            "id": 15,
+            "legend": false,
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "p90:gitlab.sql_duration_seconds{*}.as_count(), p75:gitlab.sql_duration_seconds{*}.as_count(), p50:gitlab.sql_duration_seconds{*}.as_count(), min:gitlab.sql_duration_seconds{*}.as_count(), max:gitlab.sql_duration_seconds{*}.as_count()",
```

Comment:
- ditto template variables
- Just curious, the `as_count` were added automatically? I think that's weird for a distribution metric but I'm not 100% sure how that works.

Reply:
Yes, they were added automatically